android/build.gradle

```diff
@@ -83,7 +83,7 @@ dependencies {
   // For < 0.71, this will be from the local maven repo
   // For > 0.71, this will be replaced by `com.facebook.react:react-android:$version` by react gradle plugin
   //noinspection GradleDynamicVersion
-  implementation "com.facebook.react:react-native:0.73.6"
+  implementation "com.facebook.react:react-native:+"
```

Comment:
You need to lock this to a specific version

Reply:
As mentioned in the comment, the version will be replaced. The actual version used to build the app is determined by the React Native package installed in the app. Typically, libraries use `+` because this version can vary.   But we can specify a particular version for reference.